tools/test_backups2datalad/test_commands.py

```diff
@@ -172,3 +175,53 @@ async def test_backup_zarrs(new_dandiset: SampleDandiset, tmp_path: Path) -> Non
         asset.zarr for asset in assets.values()
     )
     await new_dandiset.check_all_zarrs(ds.ds, backup_root / "zarr")
+
+
+async def test_backup_committed_zarr(
```

Comment:
note: really hard to figure what is the difference from previous test by doing visual diff since most is the same... ok, it seems to cover that codeblock so must be doing what is intended